public/app/plugins/panel/gauge/GaugePanel.tsx

```diff
@@ -12,10 +12,15 @@ export class GaugePanel extends PureComponent<PanelProps<GaugeOptions>> {
     menuProps: DataLinksContextMenuApi
   ): JSX.Element => {
     const { options } = this.props;
-    const { value, width, height } = valueProps;
+    const { width, height, count, value } = valueProps;
     const { field, display } = value;
     const { openMenu, targetClassName } = menuProps;
 
+    // hide title if only 1 gauge is being displayed and no custom displayName set
+    if (count === 1 && !field.displayName) {
+      display.title = undefined;
+    }
```

Comment:
Nit: not sure why you want to hide title in such scenario? Maybe instead we could have an option to hide the title explicitly?

This also causes some vertical misalignment in the panel:
![image](https://user-images.githubusercontent.com/2376619/86220924-88000000-bb84-11ea-9088-6d94c84452dc.png)

Reply:
this was just replicating logic that was before in getFieldDisplayValues

Reply:
strange, not seeing that missalignment, the logic in Gauge is to check for null or undefined, so it does not use any space if title is undefined